packages/app-desktop/gui/WindowCommandsAndDialogs/commands/newNote.ts

```diff
@@ -14,9 +15,16 @@ export const declaration: CommandDeclaration = {
 export const runtime = (): CommandRuntime => {
 	return {
 		execute: async (_context: CommandContext, body = '', isTodo = false) => {
-			const folderId = Setting.value('activeFolderId');
+			let folderId = Setting.value('activeFolderId');
 			if (!folderId) return;
 
+			const folder = await Folder.load(folderId);
+			if (!folder || !!folder.deleted_time) {
+				const defaultFolder = await Folder.defaultFolder();
+				if (!defaultFolder) return;
+				folderId = defaultFolder.id;
+			}
```

Comment:
The whole idea with this "activeFolderId" was to have a guaranteed, valid, target whenever a notebook is needed. It seems the concept is broken now since the introduction of the trash folder, and we don't want to have this checking logic duplicate everywhere.

Maybe an alternative would be to created a helper function like "getValidNotebook" that is guaranteed to return such a notebook?

Or maybe this command should just throw if someone attempts to create a notebook when the trash is selected?

In fact, based on `newNoteEnabledConditions`, this command is not even enabled in the trash, so maybe that's the issue, that it can be called even though it's disabled?